test/utils.ts

```diff
@@ -1,6 +1,10 @@
+// import debug from 'debug';
+import path from 'path';
+import { readFileSync } from 'fs';
 import { Keypair, PublicKey, TransactionCtorFields } from '@solana/web3.js';
 
 export const NETWORK = 'devnet';
+export const LOCAL_NETWORK = 'http://localhost:8899';
```

Comment:
This is fine, but to avoid duplication and make sure you're connecting to the correct URL you should just do `import { LOCALHOST as LOCAL_NETWORK } from '@metaplex-foundation/amman'`.